node/pkg/accountant/audit.go

```diff
@@ -277,12 +281,12 @@ func (acct *Accountant) queryMissingObservations() ([]MissingObservation, error)
 	acct.logger.Debug("acctaudit: submitting missing_observations query", zap.String("query", query))
 	respBytes, err := acct.wormchainConn.SubmitQuery(acct.ctx, acct.contract, []byte(query))
 	if err != nil {
-		return nil, fmt.Errorf("missing_observations query failed: %w", err)
+		return nil, fmt.Errorf("missing_observations query failed: %w, %s", err, query)
 	}
 
 	var ret MissingObservationsResponse
 	if err := json.Unmarshal(respBytes, &ret); err != nil {
-		return nil, fmt.Errorf("failed to parse missing_observations response: %w", err)
+		return nil, fmt.Errorf("failed to parse missing_observations response: %w, resp: %s", err, string(respBytes))
```

Comment:
this would not render properly if it's just bytes but seems ok here

Reply:
It's json, so I think this is fine.